bookkeeper-server/src/test/java/org/apache/bookkeeper/replication/TestReplicationWorker.java

```diff
@@ -1156,11 +1160,49 @@ public void testRWShutDownInTheCaseOfZKOperationFailures() throws Exception {
 
     @Test
     public void testRepairedNotAdheringPlacementPolicyLedgerFragmentsOnRack() throws Exception {
-        testRepairedNotAdheringPlacementPolicyLedgerFragments(RackawareEnsemblePlacementPolicy.class);
+        testRepairedNotAdheringPlacementPolicyLedgerFragments(RackawareEnsemblePlacementPolicy.class, null);
+    }
+
+    @Test
+    public void testReplicationStats() throws Exception {
+        BiConsumer<Boolean, ReplicationWorker> checkReplicationStats = (first, rw) -> {
+            try {
+                final Method rereplicate = rw.getClass().getDeclaredMethod("rereplicate");
+                rereplicate.setAccessible(true);
+                final Object[] objects = new Object[0];
+                final Object result = rereplicate.invoke(rw, objects);
```

Comment:
We can use `rereplicate.invoke(rw);` directly

Reply:
@horizonzy Thanks. I have addressed it.